lib/config.js

```diff
@@ -1,99 +1,40 @@
 'use strict';
 const Path = require('path');
-const Yargs = require('yargs');
+
+let defaultConfig = {
+  pg: null, // "postgres://username:password@host:port/dbname",
+  smtpUser: null,
+  smtpPass: null,
+  smtpHost: 'smtp.gmail.com',
+  smtpFrom: 'SpaceKit <spacekit.io@gmail.com>',
+  awsHostedZoneId: null,
+  awsAccessKeyId: null,
+  awsSecretAccessKey: null,
+  awsRecordSetTtl: 1,
+  api: 'api',
+  web: 'www',
+  host: '127.0.0.1.nip.io',
```

Comment:
I remember why I wanted this here: it's so that we can just run the spacekit binary to test, without any additional configuration.